Assets/MRTK/Core/Interfaces/TeleportSystem/IMixedRealityTeleportHotspot.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Teleport
+{
+    public interface IMixedRealityTeleportHotspot
+    {
+        /// <summary>
+        /// The position the teleport will end at.
+        /// </summary>
+        Vector3 Position { get; }
+
+        /// <summary>
+        /// The normal of the teleport raycast.
+        /// </summary>
+        Vector3 Normal { get; }
+
+        /// <summary>
+        /// Is the teleport target active?
+        /// </summary>
+        bool IsActive { get; }
+
+        /// <summary>
+        /// Should the target orientation be overridden?
+        /// </summary>
+        bool OverrideTargetOrientation { get; }
+
+        /// <summary>
+        /// Should the destination orientation be overridden?
+        /// Useful when you want to orient the user in a specific direction when they teleport to this position.
+        /// </summary>
+        /// <remarks>
+        /// Override orientation is the transform forward of the GameObject this component is attached to.
+        /// </remarks>
+        float TargetOrientation { get; }
```

Comment:
I think I am confused by the doccomment, variable name and the type. From the comment and variable name it seems that it should be a Vector3 since it is "orientation"? And yet the first sentence in the summary section seems to suggest that this should be a bool also. The remarks section also looks confusing to me.

Reply:
Going through the use cases of this variable, I found this

        [Tooltip("The Y orientation of the pointer - used for rotation and navigation")]

Going to update the naming + summaries here.